install/tools/setupIstioVM.sh

```diff
@@ -31,8 +31,10 @@ function istioVersionSource() {
 # Will use the generated "kubedns" file.
 function istioNetworkInit() {
   if [[ ! -r /etc/dnsmasq.d ]] ; then
-    apt-get update
-    sudo apt-get -y install dnsmasq
+    echo "*** Running apt-get update..."
+    apt-get update > /dev/null
+    echo "*** Running apt-get install dnsmasq..."
+    apt-get --no-install-recommends -y install dnsmasq
```

Comment:
We don't need this 'sudo' anymore?

Reply:
the whole thing is running as root; if sudo was needed for apt-get install it would have been needed for apt-get update